src/main/kotlin/no/nav/familie/ef/personhendelse/client/OppgaveClient.kt

```diff
@@ -33,7 +39,23 @@ class OppgaveClient(
                 opprettOppgaveRequest,
                 HttpHeaders().medContentTypeJsonUTF8()
             )
-        return response.data?.oppgaveId
+        return response.data!!.oppgaveId
```

Comment:
Tenker vi burde kunne bruke `getDataOrThrow()` her også i stedet for `!!`? 
```suggestion
        return response.getDataOrThrow().oppgaveId
```